doc/guides/writing-tests.md

```diff
@@ -209,6 +209,35 @@ const assert = require('assert');
 const freelist = require('internal/freelist');
 ```
 
+### Assertions
+
+When writing assertions for comparison, prefer the strict versions:
```

Comment:
Nit: Probably clearer to have singular/plural agreement: `writing an assertion for comparison` or `writing assertions for comparisons`.